test/truefi2/lines-of-credit/TrueFiCreditOracle.test.ts

```diff
@@ -35,6 +36,11 @@ describe('TrueFiCreditOracle', () => {
         .to.be.revertedWith('Ownable: caller is not the owner')
     })
 
+    it('cannot be set to address(0)', async () => {
```

Comment:
IMO would be cleaner, but not sure.
```suggestion
    it('cannot be set to zero address', async () => {
```